src/main/resources/com/google/api/codegen/java/smoke_test.snip

```diff
@@ -11,6 +11,11 @@
     private static final String PROJECT_ENV_NAME = "GOOGLE_CLOUD_PROJECT";
     private static final String LEGACY_PROJECT_ENV_NAME = "GCLOUD_PROJECT";
 
+    @@Test
+    public void run() {
+      main(null);
+    }
```

Comment:
Will smoke tests run when we do `./gradlew test`? It seems like that would make api-client-staging fail, right? (since a project wouldn't be defined)

Reply:
It will. And it will fail unless you have an env var to denote project_id.  I now configured the build script so that smoke tests are excluded when running `./gradlew test`.